src/components/helpwin/HelpWin.vue

```diff
@@ -35,13 +35,14 @@
       icon: {type: String, required: false, default: 'help'}
     },
     data () {
+      let config = this.$appConfig.modules['wgu-helpwin'] || {};
```

Comment:
Somehow unrelated to the PR goal, correct?

Reply:
Possibly not: as defaults (AFAIR) were not in place.

Reply:
OK, thanks!